website/conferences/model.py

```diff
@@ -27,19 +27,18 @@ class Conference(StoredObject):
     talk = fields.BooleanField(default=True)
     # field_names are used to customize the text on the conference page, the categories
     # of submissions, and the email adress to send material to.
-    field_names = fields.DictionaryField(
-        default=lambda: {
-            'submission1': 'poster',
-            'submission2': 'talk',
-            'submission1_plural': 'posters',
-            'submission2_plural': 'talks',
-            'meeting_title_type': 'Posters & Talks',
-            'add_submission': 'poster or talk',
-            'mail_subject': 'Presentation title',
-            'mail_message_body': 'Presentation abstract (if any)',
-            'mail_attachment': 'Your presentation file (e.g., PowerPoint, PDF, etc.)'
-        }
-    )
+    DEFAULT_FIELD_NAMES = {
+        'submission1': 'poster',
+        'submission2': 'talk',
+        'submission1_plural': 'posters',
+        'submission2_plural': 'talks',
+        'meeting_title_type': 'Posters & Talks',
+        'add_submission': 'poster or talk',
+        'mail_subject': 'Presentation title',
+        'mail_message_body': 'Presentation abstract (if any)',
+        'mail_attachment': 'Your presentation file (e.g., PowerPoint, PDF, etc.)'
+    }
+    field_names = fields.DictionaryField(default=DEFAULT_FIELD_NAMES)
```

Comment:
Had to move the `DEFAULT` outside of the class definition for that tow work. Corrected associated instances of variable use.